common/models/role.js

```diff
@@ -240,6 +241,15 @@ module.exports = function(Role) {
     Role.isAuthenticated(context, callback);
   });
 
+  Role.registerResolver(Role.EVERYONE_PROMISE, function(role, context) {
```

Comment:
Sure, sorry I didn't notice that it's also in role.js, it should only be defined in test.